forge-gui/res/cardsfolder/upcoming/spawning_pod.txt

```diff
@@ -0,0 +1,10 @@
+Name:Spawning Pod
+ManaCost:2 G
+Types:Artifact
+A:AB$ ChangeZone | Cost$ 1 T Sac<1/Creature> | Origin$ Library | Destination$ Battlefield | AtRandom$ True | NoShuffle$ True | Mandatory$ True | NoLooking$ True | NoReveal$ True | RememberChanged$ True | ChangeType$ Creature.cmcEQX | ChangeNum$ 1 | SorcerySpeed$ True | AILogic$ SacAndUpgrade | SubAbility$ DBAnimate | SpellDescription$  Seek a creature card with mana value equal to 1 plus the sacrificed creature's mana value and put that card onto the battlefield. That creature is a Phyrexian in addition to its other types. Activate only as a sorcery.
+SVar:DBAnimate:DB$ Animate | Defined$ Remembered | Types$ Phyrexian | Duration$ Permanent | SubAbility$ DBCleanup
```

Comment:
it probably needs to be AnimateSubAbility so it would enter the battlefield as Phyrexian

Reply:
RememberChanged is not needed